psi4/driver/driver_cbs.py

```diff
@@ -917,135 +1137,468 @@ def return_energy_components():
+        sup = build_superfunctional_from_dictionary(functionals[funcname], 1, 1, True)[0]
+        alpha = alphadata[0] + alphadata[1] * sup.x_alpha() + alphadata[2] * sup.c_alpha()
+        return alpha
+    elif xtpl_type == "dh":
+        if bzetas == [2, 3]:
+            return 2.2
+        elif bzetas == [3, 4]:
+            return 3.0
+        else:
+            raise ValidationError(f"DFT dh extrapolation alpha for basis sets {bnames} undefined.")
+
+
+def _get_default_alpha(
+    xtpl_type: Union[str, None], 
+    bdata: Tuple[List[str], List[int]], 
+    funcname: str = None
+) -> Union[float, None]:
+    """ 
+    A helper function to determine default extrapolation alpha. Currently only
+    used by :py:func:`_interpret_cbs_inputs` to get default alphas for DFA, as both
+    "scf" and "corl" types have their default alpha set.
+
+
+    Parameters
+    ----------
+    xtpl_type
+        {'scf', 'corl', 'fctl', 'dh', None}
+        Extrapolation type: 'scf' and 'fctl' for the total energy, 'corl' and 'dh'
+        for just the correlation component, None if no extrapolation requested.
+    bdata
+        Names and zetas of the basis sets to extrapolate from.
+    funcname
+        Functional name for DFT extrapolation.
+
+    Returns
+    -------
+    alpha : Union[float, None]
+        Extrapolation alpha to be used.
+    """
+    nbasis = len(bdata[0])
+    if nbasis == 1:
+        return None
+    elif nbasis == 2:
+        if xtpl_type == "scf":
+            return 1.63  # this is the scf_xtpl_helgaker_2 default
+        elif xtpl_type == "corl":
+            return 3.00  # this is the corl_xtpl_helgaker_2 default
+        elif xtpl_type in ["fctl", "dh", "dft"]:
+            return _get_dfa_alpha(xtpl_type, bdata, funcname)
+        else:
+            raise ValidationError(
+                f"Stage treatment must be one of ['scf','corl','fctl','dh',"
+                f"'dft'], not '{xtpl_type}'"
+            )
+    elif nbasis == 3:
+        return None  # we don't need alpha for _3 methods
+
+
+def _interpret_cbs_inputs(
+    cbs_metadata: List[dict], 
+    molecule: Union[qcdb.molecule.Molecule, core.Molecule],
+) -> List[dict]:
     """ A helper function which validates the ``cbs_metadata`` format,
     expands basis sets, and provides sensible defaults for optional arguments.
 
+    There are currently three ways to run a CBS calculation:
+
+    1) By specifying arguments, such as ``scf_wfn``, ``corl_basis``, or ``delta_scheme``.
+       This is the original interface, however option (3) is preferred nowadays.
+    2) By using the "method/basis+delta" syntax. This option offeres limited features,
+       as the default treatments and alphas are imposed.
+    3) By explicitly passing ``cbs_metadata`` as one of the arguments. This is the
+       full-featured user-facing interface.
+
+    These three are interpreted into a single, internal specification here. For the
```

Comment:
I read this docstring as saying that the input _could_ be in form (1), (2), or (3). Is this correct? I assumed that (1) and (2) had been converted to (3), which this function would then convert into the internal specification. If so, then this docstring is incorrect.